rust/connlib/libs/common/src/session.rs

```diff
@@ -55,6 +56,10 @@ pub trait ControlSession<T, CB: Callbacks> {
 /// A session is created using [Session::connect], then to stop a session we use [Session::disconnect].
 pub struct Session<T, U, V, R, M, CB: Callbacks> {
     runtime_stopper: tokio::sync::mpsc::Sender<StopRuntime>,
+    // The guard must not be dropped before the runtime is dropped, otherwise logs won't get
+    // flushed to the logfile.
+    #[allow(dead_code)]
```

Comment:
Just prefix it with `_` so you don't need to allow `dead_code`.

Reply:
D'oh, thanks!